# Mensaje para una caja de detecciÃ³n individual

string class_name
float32 probability
float32 xmin
float32 ymin
float32 xmax
float32 ymax
